cni/pkg/ambient/net.go

```diff
@@ -18,18 +18,25 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io/fs"
 	"net"
 	"net/netip"
 	"os"
+	"path"
+	"path/filepath"
+	"runtime"
 	"strings"
 
 	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
```

Comment:
```suggestion
	"github.com/vishvananda/netns"
```

Note that in `tools/istio-iptables/pkg/capture/run_linux.go` we use 

`"github.com/containernetworking/plugins/pkg/ns"` - which does exactly the same thing as `"github.com/vishvananda/netns"`

We should pick one or the other and use that consistently everywhere.

Reply:
Updated with the consistent one 'github.com/containernetworking/plugins/pkg/ns'